src/commcare_cloud/ansible/roles/kafka/tasks/main.yml

```diff
@@ -56,7 +56,7 @@
   when: ansible_distribution_version == '14.04'
 
 - name: Install Kafka systemd script
-  template: src=kafka.server.j2 dest=/etc/systemd/system/kafka.service mode=0755
+  template: src=kafka-server.j2 dest=/etc/systemd/system/kafka.service mode=0755
```

Comment:
Oh I think I made a mistake. This should be kafka.service.j2. Making that change and deploying it

Reply:
@danny I ran ansible playbook from this pr and it is fine